torchrl/envs/libs/vmas.py

```diff
@@ -115,13 +115,20 @@ class VmasWrapper(_EnvWrapper):
         env (``vmas.simulator.environment.environment.Environment``): the vmas environment to wrap.
 
     Keyword Args:
-        num_envs (int): Number of vectorized simulation environments.
-        device (torch.device, optional): Device for simulation.
-        continuous_actions (bool, optional): Weather to use continuous actions. Defaults to ``True``.
-        max_steps (int, optional): Maximum number of steps in each vectorized environment after which done is returned.
-            Defaults to ``None`` (no truncation).
+        num_envs (int): Number of vectorized simulation environments. VMAS perfroms vectorized simulation using PyTorch.
+            This argument decides the number of vectorized envuironments that should be simulated in a batch. It will also
+            determine the batch size of the environment.
+        device (torch.device, optional): Device for simulation. Defaults to ``"cpu"``. All the tensors created by VMAS
+            will be placed on this device.
+        continuous_actions (bool, optional): Whether to use continuous actions. Defaults to ``True``. If ``False``, actions
+            will be discrete. The number of actions and their size will depend on the scenario chosen.
+            See the VMAS repositiory for more info.
+        max_steps (int, optional): Horizon of the task. Defaults to ``None`` (infinite horizon). Each VMAS scenario can
```

Comment:
just to make sure: even when rewards can be consumed there is no max_steps predefined right? All envs are not truncated?

Reply:
see the answer on the other question on this line